core/templates/dev/head/pages/base.html

```diff
@@ -119,6 +119,10 @@
       {% endif %}
     </script>
 
+    <!-- Note to developers: This script needs to be loaded before external
```

Comment:
Hi @vojtechjelinek -- I had a quick chat with @BenHenning about this. I'd forgotten about our plans to try and remove base.html, but this is actually separate from removing Jinja. So, do you think you could put the script together with pages/header_js_libs.html instead? That way, we don't have the delay on page load, and we also resolve the issues with possible duplication due to pages (like the admin page) that don't depend on base.html. I've checked with @BenHenning and he is also happy with this.

Thanks!